lib/WebDriverElement.php

```diff
@@ -132,4 +132,12 @@ public function submit();
      * @return string The opaque ID.
      */
     public function getID();
+
+    /**
+     * Take screenshot of a specific element.
+     *
+     * @param string $save_as The path of the screenshot to be saved.
+     * @return string The screenshot in PNG format.
+     */
+    public function takeElementScreenshot($save_as = null);
```

Comment:
Extending interface is something we unfortunately can't do in 1.x - because if would break backwards compatibility.

So please comment the line and add some todo like this:

```
    /**
     * Take screenshot of a specific element.
     *
     * @param string $save_as The path of the screenshot to be saved.
     * @return string The screenshot in PNG format.
     * @todo Add in next major release (BC)
     */
    //public function takeElementScreenshot($save_as = null);
```
We will then uncomment it in 2.0.

Reply:
Done